cms/search/utils.py

```diff
@@ -16,12 +16,14 @@ def build_standard_resource_dict(page: "Page") -> dict:
     """Returns a dict with the standard resource fields.
     This covers the non-release case (and also forms the base of the release case).
     """
+    title = getattr(page, "display_title", page.title)
+
     return {
         "uri": build_page_uri(page),
         "content_type": page.search_index_content_type,
         "release_date": (page.release_date.isoformat() if getattr(page, "release_date", None) else None),
         "summary": get_text_for_indexing(force_str(page.summary)),
-        "title": page.title,
+        "title": title,
```

Comment:
Nit: inline it?
```py
        "title": getattr(page, "display_title", page.title),
```

Reply:
[7b02e91](https://github.com/ONSdigital/dis-wagtail/pull/349/commits/7b02e91408d85a7db5be340e7c2edbed305c75c0)